src/daemon.cpp

```diff
@@ -159,10 +172,10 @@ bool Daemon::parseConfig(const QJsonObject& obj, Config& config) {
         continue;
       }
 
-      allowedIPAddressRanges.append(
-          QString("%1/%2").arg(address.toString()).arg(range.toInt(0)));
+      config.m_allowedIPAddressRanges.append(IPAddressRange(
```

Comment:
No, it's related. In order to support the linux server-switching, we need to have the allowIPAddressRange in an "object" form instead of "strings". So, I had to change `m_allowedIPAddressRanges` from being `QStringList` to `QList<IPAddressRange>`.